lib/elixir/pages/getting-started/keywords-and-maps.md

```diff
@@ -33,14 +33,14 @@ We can also use options to limit the splitting algorithm to a maximum number of
 
 ```elixir
 iex> String.split("1  2  3", " ", [trim: true, parts: 2])
```

Comment:
I wonder if we should pick another example now? Although the only other option that comes to mind is `inspect`.

Reply:
Another option is to invert the examples: first we show the parts option and then we show the trimming one. It may be more logical.

Reply:
how about [to_timeout](https://hexdocs.pm/elixir/main/Kernel.html#to_timeout/1-examples)?
it's in Kernel, and very easy to understand

```elixir
to_timeout(hour: 1, minute: 30)
5400000
```

Reply:
Oh but people who installed an old version might not have it

Reply:
On the other hand it requires additional context on where timeouts are used? There is always `Date.shift` though.

Reply:
Hrm... now I am worried the representation of newlines may cause confusion for a newcomer. :(

So I guess we either keep this one and invert the order, since parts are processed first, or use `Date.shift`. :)

Reply:
Problem with `Date.shift` is that it would introduce sigils and structs at a very early stage of the getting started.
Trying to think of something intuitive...

Reply:
> On the other hand it requires additional context on where timeouts are used? There is always Date.shift though.

If we assume people have a recent enough version of Elixir, I think the context can be pretty minimal and the examples natural enough. We could say something like

> Various functions in Elixir or Erlang libraries accept a timeout in milliseconds. For these cases, we can use the `to_timeout` helper:
>
> ```elixir
> iex> to_timeout([minute: 1])
> 60000
> 
> iex> to_timeout([minute: 1, second: 30])
> 90000
> 
> iex> to_timeout(minute: 1, second: 30)
> 90000
> ```

Reply:
Another example using just primitive types:

```elixir
iex(1)> Base.encode32("foobar")
"MZXW6YTBOI======"
iex(2)> Base.encode32("foobar", case: :lower)
"mzxw6ytboi======"
iex(3)> Base.encode32("foobar", case: :lower, padding: false)
"mzxw6ytboi"
```

Reply:
Let’s try keeping as is but different order. Do you want me to push something?

Reply:
If that doesn’t work, I think Base can be a good second option.

Reply:
> Let’s try keeping as is but different order. 

Sorry I didn't get it 😅

> Do you want me to push something?

Yes please 🙏